packages/vhd-lib/merge.js

```diff
@@ -116,8 +115,8 @@ module.exports = limitConcurrency(2)(async function merge(
         concurrency,
       }
     )
+    clearInterval(mergeStateInterval)
```

Comment:
Use either [`golike-defer`](https://github.com/JsCommunity/golike-defer) or `try/finally` to make sure the interval is always cleared.

Reply:
Another, maybe simpler, solution, could be to simply throttle this function :slightly_smiling_face: